trimmer.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.strip
+  end
```

Comment:
- [ ] The `correct_name` method here in the `TrimmerDecorator` should make sure that the output of `@nameable.correct_name` has a maximum of 10 characters. If it's longer it should trim the word. You can use the `.length` property and create a conditional statement to check if the output has a maximum of 10 characters. If it's longer it should trim the word.